gui/default/syncthing/device/shareDeviceDialogView.html

```diff
@@ -0,0 +1,52 @@
+<modal id="share-device-confirmation" status="warning" icon="{{shareDeviceParams.icon}}" heading="{{shareDeviceParams.heading}}" large="no" closeable="yes">
+  <div class="modal-body">
+    <p>
+      <span translate>The following content will automatically be inserted into a message.</span>
+      <span ng-switch="shareDeviceParams.type">
+        <span ng-switch-when="email">
+          <span translate>You should then use your own email account to send it.</span>
+          <span translate>Proceed only if you have a registered email client.</span>
+        </span>
+        <span ng-switch-when="sms">
+          <span translate>You should then use your own phone number to send it.</span>
+          <span translate>Proceed only if you have a registered SMS client.</span>
+        </span>
+      </span>
+      <span translate>Alternatively, copy the content to clipboard and send it manually yourself.</span>
```

Comment:
Do you think it would be fine to remove the word "clipboard" from the button text entirely and just leave "copy" instead? It can still stay in the code, of course, but the truth is that a typical computer user has zero idea what "clipboard" actually means.

Reply:
I guess so, neither Discourse nor Github seems to use it based on my quick checks ;-)

Reply:
Renamed to just "copy". It's still called "copyToClipboard" in the code itself.